op-devstack/dsl/l2_el.go

```diff
@@ -215,3 +223,19 @@ func (el *L2ELNode) PayloadByNumber(number uint64) *eth.ExecutionPayloadEnvelope
 	el.require.NoError(err, "failed to get payload")
 	return payload
 }
+
+func (el *L2ELNode) NewPayload(ref *L2ELNode, number uint64) *NewPayloadResult {
```

Comment:
Nit: We generally use `ref` for a block reference, not for nodes. Can we update the second node to `el2` or `second` or something other than `ref`?

```suggestion
func (el *L2ELNode) NewPayload(el2 *L2ELNode, number uint64) *NewPayloadResult {
```

Reply:
Used `refNode` and also added godoc at https://github.com/ethereum-optimism/optimism/pull/17714/commits/cb3dfe13812b8e3417a6f47d804b045d547ea689